dags/taar_amodump.py

```diff
@@ -33,3 +33,21 @@
     output_visibility="private",
     dag=dag
 )
+
+amowhitelist = EMRSparkOperator(
+    task_id="taar_amowhitelist",
+    job_name="Transform the AMO addon JSON into a whitelisted set of addons for futher TAAR processing",
```

Comment:
This job name should be short, its the identifier in the EMR console. Maybe something like "Update TAAR AMO-whitelist"

Reply:
Thanks, I've shortened the job_name now.